### PR TITLE
Territory control rework

### PR DESCRIPTION
**Pull request by magicksRampage**
Territory Control Rework
-decay time is now proportional to capture time
-capture speed increases with current capture progress
-capture speed decreases with number of friendly adjacent districts
-districts that are fully surrounded by friendly districts cannot be captured and don't decay
-kingpin slime generated by districts increases for each friendly adjacent district

Zero Visibility/Fog of Endless War
-Discord roles no longer tell you a player's location
-instead you can !scout adjacent districts and subzones to see how many gangsters there are

Misc
-posting in the server no longer prevents you from being kicked out of subzones
-you can't act for 10 seconds after returning from offline mode

### CODE REVIEW

File: ewcfg.py [hunk 1]
```diff
@@ -3377,6 +3411,39 @@ def wef_scythe(ctn = None):
 	if poi.is_capturable:
 		capturable_districts.append(poi.id_poi)
 
+# maps districts to their immediate neighbors
+poi_neighbors = {
```
Comment (krakissi) [inline ewcfg.py]: This is a maintenance nightmare. We should use the map array to discover neighbor districts and construct this dictionary at server startup.

Comment (magicksRampage) [reply ewcfg.py]: it sure seemed like just listing them by hand would be less work than writing a method to do it automatically. but looking at it, the path_to method already does that, if you don't pass an end point, doesn't it?

Comment (magicksRampage) [reply ewcfg.py]: it should work now, but I can't actually run the code currently, so fingers crossed, lol

Comment (magicksRampage) [reply ewcfg.py]: should work now

**Pull request merged**